README.md

```diff
@@ -163,40 +166,16 @@ A JSON object to describe the profilers to be used and their arguments. Below ar
     }
   }
 ```
+Out of the box, AR contains the plugins listed below which can immediately be used as a profiler for an experiment.
 
-```js
-  "profilers": {
-    "batterystats": {
-      "cleanup": true,
-      "subject_aggregation": "default",
-      "experiment_aggregation": "default",
-      "enable_systrace_parsing": true,
-      "python2_path": "python2"
-    }
-  }
-```
-
-```json
-  "profilers": {
-    "Garbagecollection": {
-      "subject_aggregation" : "default"
-    }
-  }
-```
-The garbage collection (GC) plugin gathers and counts GC log statements by searching in adb's logcat for logs that meet the format of a GC call as described [here](https://dzone.com/articles/understanding-android-gc-logs).
-The default subject aggregation lists the counted GC calls in a single file for easy further processing.
-```json
-  "profilers": {
-    "Frametimes": {
-      "subject_aggregation" : "default",
-      "sample_interval": 1000
-    }
-  }
-```
-The frame times plugin gathers unique frame rendering durations (in nanoseconds) by utilizing `dumpsys gfxinfo framestats` and counts the amount of delayed frames that occurred following the 16ms threshold [defined by Google](https://developer.android.com/training/testing/performance).
-The sample interval is configurable but advised to keep under 120 seconds as the framestats command returns only data from frames rendered in the past 120 seconds as described [here](https://developer.android.com/training/testing/performance).
-Shorter sample intervals will not cause duplication in the frames gathered as only unique frames are kept.
-The default subject aggregation consists of combining both the frametimes as the delayed frames count in single files for easy further processing.
+| Name (quality)                                                                       | Description                                                                                                                                                                                                                                                                                 |
```

Comment:
The title of the first column should be `Name (quality attribute)`

Reply:
Fixed it

Reply:
Fixed